modules/tinymce/src/plugins/autosave/main/ts/ui/Buttons.ts

```diff
@@ -1,9 +1,9 @@
 import Editor from 'tinymce/core/api/Editor';
-import { Toolbar } from 'tinymce/core/api/ui/Ui';
+import { Menu } from 'tinymce/core/api/ui/Ui';
 
 import * as Storage from '../core/Storage';
 
-const makeSetupHandler = (editor: Editor) => (api: Toolbar.ToolbarButtonInstanceApi) => {
+const makeSetupHandler = (editor: Editor) => (api: Menu.MenuItemInstanceApi) => {
```

Comment:
This change seems unexpected. `Menu.MenuItemInstanceApi` sounds like the API of a single menu item, whereas this is in Buttons. Is this really intended?